.pylintrc

```diff
@@ -142,6 +142,7 @@ disable=print-statement,
 # Not disabled in default config
         broad-except,
         consider-using-get,
+        dangerous-default-value,
```

Comment:
FYI it was complaining about default list initialization:

```python
# If `f` mutates `vals` this may lead to undesired behavior.
def f(vals: List = []):
    pass
```

This is an interesting lint but let's stick to our decision of avoiding pain by default ^^